applications/luci-app-isc-dhcp/Makefile

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2022 Jaymin Patel <jem.patel@gmail.com>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI support for the ISC DHCP
+LUCI_DEPENDS:=+luci-base isc-dhcp-server-ipv4 +isc-dhcp-dyndns-ipv4
```

Comment:
Please remove the redundant `isc-dhcp-server-ipv4` dependency (the one without `+`)

Reply:
corrected dependencies.